dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -3630,6 +3630,69 @@ public void VerifyHeaderContentMarginOnTopNav()
             }
         }
 
+        [TestMethod]
+        [TestProperty("TestSuite", "D")]
+        public void VerifyHeaderContentMarginOnMinimalNav()
+        {
+            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView Test" }))
+            {
+                Log.Comment("Change display mode to left minimal");
+                var panelDisplayModeComboBox = new ComboBox(FindElement.ByName("PaneDisplayModeCombobox"));
+                panelDisplayModeComboBox.SelectItemByName("LeftMinimal");
+                Wait.ForIdle();
+
+                Log.Comment("Get HeaderContent Margin");
+                var getHeaderContentMarginButton = new Button(FindElement.ByName("GetHeaderContentMargin"));
+                getHeaderContentMarginButton.Invoke();
+                Wait.ForIdle();
+
+                var result = new TextBlock(FindElement.ByName("HeaderContentMarginResult"));
+                Verify.AreEqual(result.GetText(), "8,5,0,0");
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("TestSuite", "D")]
+        public void VerifyCustomHeaderContentMarginOnTopNav()
+        {
+            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView custom ThemeResources Test" }))
+            {
+                Log.Comment("Flipping orientation: Left -> Top.");
+                var flipOrientationButton = new Button(FindElement.ByName("FlipOrientationButton"));
+                flipOrientationButton.Invoke();
+                Wait.ForIdle();
+
+                Log.Comment("Get HeaderContent Margin");
+                var getHeaderContentMarginButton = new Button(FindElement.ByName("GetHeaderContentMargin"));
+                getHeaderContentMarginButton.Invoke();
+                Wait.ForIdle();
+
+                var result = new TextBlock(FindElement.ByName("HeaderContentMarginResult"));
+                Verify.AreEqual(result.GetText(), "16,3,0,7");
+            }
+        }
+
+        [TestMethod]
+        [TestProperty("TestSuite", "D")]
+        public void VerifyCustomHeaderContentMarginOnMinimalNav()
```

Comment:
Can you move these tests into the NugetRelease tests? There's enough differences with how the resources are included in the test app versus a nuget-consuming app that I want to just be sure that these overrides work and to test them in a "real" scenario instead.